src/test/scala/org/broadinstitute/dsde/workbench/sam/service/ResourceServiceSpec.scala

```diff
@@ -760,4 +760,26 @@ class ResourceServiceSpec extends FlatSpec with Matchers with ScalaFutures with
 
     service.listAllFlattenedResourceUsers(resource).unsafeRunSync() should contain theSameElementsAs Set(dummyUserIdInfo, user1, user2, user3, user4, user5, user6)
   }
+
+  "loadAccessPolicyWithEmails" should "get emails for users, groups and policies" in {
+    service.createResourceType(defaultResourceType).unsafeRunSync()
+
+    val testGroup = dirDAO.createGroup(BasicWorkbenchGroup(WorkbenchGroupName("mygroup"), Set.empty, WorkbenchEmail("group@a.com"))).unsafeRunSync()
```

Comment:
minor, would be nice if you put these calls in a for comprehension and then has once `futureValue` or `unsafeRunSync`